### PR TITLE
fix #35 by completing height filters

### LINKED ISSUE #35: complete height filter in blocks and boxes resolver
Adding `minHeight` and `maxHeight` to resolvers would help more powerful filters throughout the code.

### PR DESCRIPTION
**Pull request by SepehrGanji**

### CODE REVIEW

File: src/graphql/resolvers/block-resolver.ts [hunk 1]
```diff
@@ -11,7 +11,10 @@ class BlockQueryArgs extends PaginationArguments {
   headerId?: string;
 
   @Field(() => Int, { nullable: true })
-  height?: number;
```
Comment (capt-nemo429) [inline src/graphql/resolvers/block-resolver.ts:14]: In order to allow finding blocks by a specific height, I think we need to keep height here alongside with min and max heights.

**Review by capt-nemo429 — CHANGES_REQUESTED** [review]

File: src/graphql/resolvers/block-resolver.ts (hunk 1 above)
Comment (SepehrGanji) [reply src/graphql/resolvers/block-resolver.ts:14]: I had something different on my mind: To find a block of a specific height, one must specify both min and max height equal to the desired height.
I think that makes it more complicated. I'll ever the code.
Let me know if you agree with my previous thought.

Comment (capt-nemo429) [reply src/graphql/resolvers/block-resolver.ts:14]: You have a good point, although I think having a `height` param is something that makes queries a bit more straight forward, if you want to fetch a single block.

Comment (SepehrGanji) [reply src/graphql/resolvers/block-resolver.ts:14]: I agree.
If everything else looks fine, we can do the merge )

**Pull request merged**